src/components/Disclosure/DisclosureDetails/DisclosureDetails.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import {CSSTransition} from 'react-transition-group';
+
+import {getCSSTransitionClassNames} from '../../utils/transition';
+import {b} from '../Disclosure';
```

Comment:
It is not good that we have bidirectional relation here. Children of `Disclosure` should not import anything from it parent except types. It can generate circular dependency